packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -156,16 +167,25 @@ contract Reserve is
     emit TobinTaxReserveRatioSet(value);
   }
 
-  /**
-   * @notice Set the ratio of reserve that is spendable per day.
-   * @param ratio Spending ratio as unwrapped Fraction.
-   */
   function setDailySpendingRatio(uint256 ratio) public onlyOwner {
     spendingRatio = FixidityLib.wrap(ratio);
     require(spendingRatio.lte(FixidityLib.fixed1()), "spending ratio cannot be larger than 1");
     emit DailySpendingRatioSet(ratio);
   }
 
+  /**
+   * @notice Set the ratio of reserve for a given ERC20 token that is spendable per day.
+   * @param erc20SpendingRatio Spending ratio as unwrapped Fraction.
+   */
+  function setDailySpendingRatioForErc20Tokens(
+    address erc20TokenAddress,
+    uint256 erc20SpendingRatio
+  ) public onlyOwner {
+    erc20DailySpendingRatio[erc20TokenAddress] = FixidityLib.wrap(erc20SpendingRatio);
+    require(spendingRatio.lte(FixidityLib.fixed1()), "spending ratio cannot be larger than 1");
+    emit DailySpendingRatioForErc20TokenSet(erc20TokenAddress, erc20SpendingRatio);
+  }
+
```

Comment:
I wonder whether CELO should be treated just as any other ERC20 and therefore be handled in this function instead of the function above.